src/main/resources/assets/memoryusagescreen/lang/ru_ru.json

```
@@ -1,8 +1,8 @@
 {
-  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
-  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
-  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
-  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Режим переключения отображения по нажатию",
+  "text.autoconfig.memoryusagescreen.title": "Конфигурация Memory Usage Screen",
```

"Конфигурация" в переводе программ подразумевает набор компонентов, а не параметров. Тут исключительно 
```suggestion
  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
```
Крайне распространённая ошибка.

---

Изучив вопрос. Согласен на употребление слова Настройки.